chainer/functions/pooling/max_pooling_2d.py

```diff
@@ -356,9 +357,26 @@ def max_pooling_2d(x, ksize, stride=None, pad=0, cover_all=True):
             ``pad=p`` and ``pad=(p, p)`` are equivalent.
         cover_all (bool): If ``True``, all spatial locations are pooled into
             some output pixels. It may make the output size larger.
+        return_indices (bool): If ``True``, pooling indices array is returned
+            together with the output variable. The returned indices are
+            expected for use by :func:`chainer.functions.upsampling_2d`.
+            Note that cuDNN will not be used for this function if
+            ``return_indices`` is set to ``True``, as cuDNN does not return
+            indices information.
 
     Returns:
-        ~chainer.Variable: Output variable.
+        ~chainer.Variable or tuple:
+            When ``return_indices`` is ``False`` (default), returns the output
+            variable.
+            When ``True``, returnes the tuple of the output variable and
```

Comment:
Could you fix the typo `returnes` -> `returns` ?

Reply:
Thanks, fxed.